k2s/cmd/k2s/cmd/image/tag.go

```diff
@@ -36,28 +37,21 @@ var (
 )
 
 func init() {
+	tagCmd.Flags().String(imageIdFlagName, "", "Image ID of the container image")
+	tagCmd.Flags().StringP(imageNameFlagName, "n", "", "Name of the container image including tag")
+	tagCmd.Flags().StringP(targetImageNameFlagName, "t", "", "New name of the container image including tag")
 	tagCmd.Flags().SortFlags = false
 	tagCmd.Flags().PrintDefaults()
 }
 
 func tagImage(cmd *cobra.Command, args []string) error {
```

Comment:
I think we need to check the scenario where there are two images with same id, and the user tries to tag the image using id. In this case, I observed that the command failed.

Reply:
fixed with:

```
if($foundLinuxImages.Count -ge 1 -or $foundWindowsImages.Count -ge 1) {
        $errMsg = "More than one image has the id: $Id. Please use --name to identify the image instead or delete the other image/s"
        if ($EncodeStructuredOutput -eq $true) {
            $err = New-Error -Severity Warning -Code 'two-images-found' -Message $errMsg
            Send-ToCli -MessageType $MessageType -Message @{Error = $err }
            return
        }
}
````